docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -97,3 +97,15 @@ delay before consuming or producing a new page::
       page_processing_delay = '5s'
     );
 
+.. _blackhole-sql-support:
+
+SQL support
+-----------
+
+The connector provides access to data temporarily stored in the Black Hole
```

Comment:
there is no data stored anywhere

Reply:
The Black Hole only stores binary logs and exists for measuring performance only, then?

Reply:
> exists for measuring performance only, then?

it's useful for testing too

> The Black Hole only stores binary logs 

i know nothing about binary logs. But i'm not an expert

Reply:
I would propose something like

"The connector is intended to test throughput by writing data to a null endpoint, and as such there is no data to be accessed with :ref:`read operation <sql-read-operations>` statements. The connector provides :ref:`globally available <sql-globally-available>` statements to write metadata to memory and data to the null endpoint. In addition, the connector supports the following features:"

Reply:
I think it's ultimately worth documenting that read statements are supported, but just note that by design they're not intended for use with this connector. Save for maybe reading table metadata that's been written to memory.

Reply:
> 
> 
> I would propose something like
> 
> "The connector supports :ref:`globally available <sql-globally-available>` and :ref:`read operation <sql-read-operations>` statements. In addition, the connector supports the following features:
> 
> (list of statements)
> 
> .. note:
> Since the connector writes data to a null endpoint, while read operation statements are supported most of them will always return `NULL`.
> "

I wrote something like this. Not exact wording, but will edit once I get feedback from eng.

Reply:
Can you add the note I proposed at the end there? That should help address @findepi 's ask that we call out that some statements, like INSERT, are technically supported but don't result in data that can be read later.

Reply:
Absolutely.